zeebe/broker/src/main/java/io/camunda/zeebe/broker/partitioning/startup/RaftPartitionFactory.java

```diff
@@ -41,6 +43,11 @@ public RaftPartition createRaftPartition(final PartitionMetadata partitionMetada
             .resolve("partitions")
             .resolve(partitionMetadata.id().id().toString());
     try {
+      if (FileUtil.isEmpty(partitionDirectory)) {
+        LOG.info(
+            "Root directory for partition {} is empty or does not exist, creating it if does not exist.",
```

Comment:
Would it be better to make the message more explicit as follows? 
```suggestion
            "Root directory '{}' for partition '{}' is empty or does not exist. The partition `{}` is starting with no pre-existing data.",
```